foxtrot-server/src/main/java/com/flipkart/foxtrot/server/resources/DocumentResource.java

```diff
@@ -169,33 +123,46 @@ private String preProcess(String table, Document document) {
                 if(tableVsDocuments.containsKey(tableName)) {
                     tableVsDocuments.get(tableName)
                             .add(document);
-                } else {
+                }else {
                     List<Document> tableDocuments = Lists.newArrayList(document);
                     tableVsDocuments.put(tableName, tableDocuments);
                 }
             }
-        } else {
+        }else {
             tableVsDocuments.put(table, documents);
         }
         return tableVsDocuments;
     }
 
-    private List<Document> filterValidDocuments(String table, List<Document> documents) {
-        if(ignoredEventConfigs != null && ignoredEventConfigs.containsKey(table)) {
-            for(Iterator<Document> iter = documents.listIterator(); iter.hasNext(); ) {
-                Document document = iter.next();
-                if(document.getData()
-                        .has(EVENT_TYPE)) {
-                    String eventType = document.getData()
-                            .get(EVENT_TYPE)
-                            .asText();
-                    if(eventToBeIgnored(table, eventType)) {
-                        LOGGER.info("Skipped event for table: {} with eventType: {} Document: {}", table, eventType, document);
-                        iter.remove();
-                    }
```

Comment:
Shall we add this in upstream ? This is added to send events to different tables? Splitting of tables based on eventTypes